templates/kafka_env.sh.erb

```diff
@@ -6,3 +6,4 @@ fi
 
 export JAVA_HOME="<%= scope.lookupvar('kafka::server::java_home') %>"
 export ZOOKEEPER="<%= scope.lookupvar('kafka::server::zookeeper_connect') %>"
+export BOOTSTRAP=localhost:9092
```

Comment:
Two minor things:
- Should it be BOOTSTRAP_SERVER
- Is the port ever going to be dynamically set to something different?

Reply:
I think BOOTSTRAP is preferable, a lot easier to type. i don't think we're polluting the namespace with that (`git grep BOOTSTRAP` in puppet repo doesnt have anything) we could always go for $KAFKABOOTSTRAP but i just think we should avoid special characters and length if possible

i don't think the port will change, the default kafka port is just 9092